metadata.rb

```diff
@@ -4,6 +4,9 @@
 license          'GNU Public License 3.0'
 description      'Installs/Configures zabbix_ng'
 long_description 'Installs/Configures zabbix_ng'
-version          '0.1.1'
+version          '0.1.2'
```

Comment:
Please do not change the cookbook version - I think it's usually a good idea to let the maintainer take care of this. Can you remove this from the PR?